### ISSUE TITLE
Update Nodejs handbook for new Task API

### ISSUE THREAD

**Issue by mgordel** [golemfactory/yagna-docs#133]
Split current handbook Task API examples to separate articles for python and nodejs.

https://handbook.golem.network/requestor-tutorials/task-processing-development

- [ ] Task Example 0: Hello World! - split into two articles
- [ ] Task Example 1: Simple hash cracker - add nodejs version
- [ ] Task Example 2: Hashcat on Golem - add nodejs version

**Comment by mateuszsrebrny** [comment]
Pending on @stranger80 at yagna-sdk-team#202

**Comment by lucekdudek** [comment]
@mgordel Please close this task and move its content to separated subtasks

**Comment by mgordel** [comment]
Moved to #155
